products/amazon-linux.md

```diff
@@ -35,33 +35,37 @@ releases:
     releaseLabel: 'AMI 2018.03'
     support: 2020-12-31
     eol: 2023-06-30
-    latest: "2018.03"
-    releaseDate: 2018-03-31
+    latest: "2018.03.0.20230207.0"
```

Comment:
Seems automation took up on "new" releases (#2558). Just out of precaution, @noqcks is it what you were expected ?

Reply:
Amazon doesn't really keep a release log of their linux versions anywhere except dockerhub, but this seems expected yep